lighthouse-core/test/results/sample_v2.json

```diff
@@ -320,7 +320,7 @@
                 "url": "http://localhost:10200/dobetterweb/dbw_tester.html",
                 "startTime": 185603.321221,
                 "endTime": 185603.961376,
-                "responseReceivedTime": 185603.89718499998,
+                "_responseReceivedTime": 185603.89718499998,
```

Comment:
viewing these inline is a bit annoying it's the only `_` 😆 We don't really want folks to use these though I guess

Reply:
> viewing these inline is a bit annoying

yeah, it seems like it really shouldn't be in the audit result anyways